test/python/transpiler/test_csp_layout.py

```diff
@@ -72,6 +72,32 @@ def test_3q_circuit_5q_coupling(self):
         self.assertEqual(layout[qr[2]], 4)
         self.assertEqual(pass_.property_set["CSPLayout_stop_reason"], "solution found")
 
+    def test_3q_circuit_5q_coupling_with_target(self):
+        """3 qubits in Tenerife, without considering the direction
+            qr1
+           /  |
+        qr0 - qr2 - 3
+              |   /
+               4
+        """
+        target = FakeYorktownV2().target
```

Comment:
lol, tenerife vs yorktown

Reply:
Lol, I think I copied the existing test and then realized there was no v2 version of tenerife or something and changed it to yorktown.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9263/commits/87d0d4078e6846d8ef6824c0797bee567c3423b8